portal/app/[locale]/stake/_components/manageStake/fees.tsx

```diff
@@ -5,7 +5,11 @@ import { useTranslations } from 'next-intl'
 import { getNativeToken } from 'utils/nativeToken'
 import { formatUnits } from 'viem'
 
-const Fees = function ({ estimatedFees }: { estimatedFees: bigint }) {
+interface Props {
+  estimatedFees: bigint
+}
```

Comment:
Let's prefer `Type` to be consistent with the rest of the app

```suggestion
type Props {
  estimatedFees: bigint
}
```

Reply:
Changed to type.